src/EntryPoints/Specials/SpecialPendingApprovals.php

```diff
@@ -17,7 +17,7 @@ public function __construct(
 		private readonly PendingApprovalRetriever $pendingApprovalRetriever,
 		private readonly LinkRenderer $linkRenderer
 	) {
-		parent::__construct( 'PendingApprovals' );
+		parent::__construct( 'PendingApprovals', restriction: 'manage-approvers' );
```

Comment:
Is this correct? This is saying I can only view my pending approvals if I'm allowed to manage approvers.

Reply:
Oops. Thanks!